cmd/utils/flags.go

```diff
@@ -2391,7 +2431,7 @@ func DialRPCWithHeaders(endpoint string, headers []string) (*rpc.Client, error)
 	}
 	var opts []rpc.ClientOption
 	if len(headers) > 0 {
-		var customHeaders = make(http.Header)
+		customHeaders := make(http.Header)
```

Comment:
Hm, why? I don't think we want to change upstream code unless necessary

Reply:
good catch, didn't think about it affecting upstream code, reverted